tests/testit/test-formats.R

```diff
@@ -49,6 +49,7 @@ test_format("glossa")
 test_format("ieee")
 test_format("ims")
 test_format("ims", output_options = list(journal = "aap"))
+test_format("informs", output_options = list(journal = "mksc"))
```

Comment:
Should `informs_articles()` support a `journal` argument ? currently, it has none so setting `output_options` does not work as you expect I believe. 

Is the aim to modify the `journal` Pandoc's variable you have set in YAML header ? 

For test, the default one in skeleton will be enough. If you want this value to be able to be set this way though, it needs to be done in `informs_article()`, setting a Pandoc metadata from within the function itself. 

But YAML field is also fine.

Reply:
BTW this is what creates the issue in CI

Reply:
Ah sorry, I misunderstood the use of output_options. I'll remove it.